Codeinwp/neve
# Feat add next page link

### Summary
- Render the pagination even for infinite scroll
- Hide it by wrapping it in a div with display none style
- Add rel="next" and rel="prev" on pagination next/previous links.

### Will affect visual aspect of the product
NO

### Test instructions
- In chrome inspect element, check if pagination markup is present even if infinite scroll is enabled
- Check if the pagination have the rel="next" / rel="prev" on Next/Previous https://prnt.sc/111abwr



Closes #1241.
